packages/suite/src/views/wallet/coinmarket/common/CoinmarketLayout/CoinmarketAccountTransactions/CoinmarketAccountTransactions.tsx

```diff
@@ -87,17 +87,22 @@ export const CoinmarketAccountTransactions = () => {
             {sortedAccountTransactions.length > 0 && (
                 <>
                     <Header>
-                        <StyledH2>
-                            <Translation id="TR_BUY_ACCOUNT_TRANSACTIONS" />
-                            <TransactionCount>
-                                {buyTransactions.length} <Translation id="TR_TRADE_BUYS" /> •{' '}
-                                {sellTransactions.length} <Translation id="TR_TRADE_SELLS" /> •{' '}
-                                {exchangeTransactions.length}{' '}
-                                <Translation id="TR_TRADE_EXCHANGES" /> • {spendTransactions.length}{' '}
-                                <Translation id="TR_TRADE_SPENDS" /> • {savingsTransactions.length}{' '}
-                                <Translation id="TR_TRADE_SAVINGS" />
-                            </TransactionCount>
-                        </StyledH2>
+                        <div>
```

Comment:
Is this div necessary?

Reply:
removed with styled setting Header to display flex
![Screenshot 2024-03-25 at 11 18 19](https://github.com/trezor/trezor-suite/assets/41970479/689375c1-4b32-43c7-99fd-7b8ba5b34217)